README.md

```diff
@@ -50,6 +50,40 @@ end
 
 ### Configuration
 
+#### Creating config files
+
+[`mix new <project>` no longer creates initial config files](https://github.com/elixir-lang/elixir/issues/8815),
+so you probably want to create one.  You can find the full documentation for
+[`Config` here](https://hexdocs.pm/elixir/1.13.2/Config.html).
+
+The simplest approach is to use a single config file, but if you plan to run your operator in
+different environments, it can be convenient to have different configs for each
+environment:
+
+```bash
+mkdir -p config
+echo -e "import Config\n\n# Put config here\n\nimport_config \"#{config_env()}.exs\"" > config/config.exs
+```
+
+If you want to follow the common dev, test, prod pattern, create those files now:
+
+```bash
+touch config/{dev,test,prod}.exs
+```
+
+Finally, if you need to provide runtime configuration (such as API token and
+other secrets), you should create `runtime.exs`:
```

Comment:
I think it's better to remind user to add runtime.exs to .gitigore file to avoid accidental leak of credentials.